parent/pom.xml

```diff
@@ -164,6 +164,11 @@
         <version>1.0</version>
         <scope>provided</scope>
```

Comment:
1. Is the `provided` scope really needed in the `dependencyManagement` section?
  * If it can be removed, then we should also remove the `test` scope on the `openjpa` dependency definition here (line 219).
2. Do we also need to update the dependency here as in the `engine/pom.xml`?

Reply:
Re 1: It's not needed.
Re 2: This is a dependency management section. If we still use the dependency in any inherting module, we should keep it here. If we don't use it anymore we can remove it. We have case 1, we use `persistence-api` still in engine-spring where we test in the case of Spring 3 against a Hibernate version that implements JPA 1.0.